benchmark/benchmain/main.go

```diff
@@ -456,22 +498,30 @@ func main() {
 			channelz.TurnOn()
 		}
 		if runMode[0] {
-			unaryBenchmark(startTimer, stopTimer, benchFeature, benchtime, s)
+			count := unaryBenchmark(startTimer, stopTimer, benchFeature, benchtime, s)
 			s.SetBenchmarkResult("Unary", benchFeature, results.N,
 				results.AllocedBytesPerOp(), results.AllocsPerOp(), sharedPos)
 			fmt.Println(s.BenchString())
 			fmt.Println(s.String())
+			printThroughput(count, benchFeature.ReqSizeBytes, count, benchFeature.RespSizeBytes)
 			resultSlice = append(resultSlice, s.GetBenchmarkResults())
 			s.Clear()
 		}
 		if runMode[1] {
-			streamBenchmark(startTimer, stopTimer, benchFeature, benchtime, s)
+			count := streamBenchmark(startTimer, stopTimer, benchFeature, benchtime, s)
 			s.SetBenchmarkResult("Stream", benchFeature, results.N,
 				results.AllocedBytesPerOp(), results.AllocsPerOp(), sharedPos)
 			fmt.Println(s.BenchString())
 			fmt.Println(s.String())
+			printThroughput(count, benchFeature.ReqSizeBytes, count, benchFeature.RespSizeBytes)
 			resultSlice = append(resultSlice, s.GetBenchmarkResults())
 			s.Clear()
+
+			bm.ResponseSize = benchFeature.RespSizeBytes
```

Comment:
In general, avoid package-level globals.  Also, avoid setting other packages' globals.  Let's find a way to configure the service handler itself.